src/Sylius/Bundle/CoreBundle/Resources/translations/messages.en.yml

```diff
@@ -202,6 +202,7 @@ sylius:
                 help: Timezone %timezone% is deprecated. Fix your php.ini file (list of deprecated timezones http://php.net/manual/en/timezones.others.php).
             version: PHP version
             version_recommended: Recommended PHP version
+            memory_limit: memory_limit
```

Comment:
You should add translation key only in this file with the proper message and revert adding this line in other `messages.*.yml` files

Reply:
Alright! I'll fix this tonight